docker/nginx/hazen.conf

```diff
@@ -10,6 +10,10 @@ server {
      # To disable buffering
      proxy_buffering off;
 
+     location /static/ {
+        alias /APP/hazen_reports;
```

Comment:
@Lucrezia-Cester I was thinking maybe we should create a `data/` directory and write the `uploads` and `hazen_reports/` directories to that so it's easier to `.gitignore` - what do you think?